.github/workflows/build.yml

```diff
@@ -239,7 +239,11 @@ jobs:
       - uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
       - uses: jupyterlab/maintainer-tools/.github/actions/check-links@v1
         with:
-          ignore_links: 'lite/ https://www.npmjs.com.* ^/.*'
+          ignore_links: |
+            lite/
+            https://www.npmjs.com.*
+            ^/.*
```

Comment:
Weird, this seems like it should work. This line is intended to ignore errors about absolute path links. But those errors are back now. :thinking: